views/components/Drawer.jsx

```diff
@@ -15,8 +15,10 @@ import Person from '@material-ui/icons/Person';
 import ColorLens from '@material-ui/icons/ColorLens';
 import ExitToApp from '@material-ui/icons/ExitToApp';
 import Settings from '@material-ui/icons/Settings';
+import { ShowChart } from '@material-ui/icons';
 
 import basePathConfig from '../../server/configs/basePathConfig';
+import { dark_sky_blue } from '../constants/styles.constants';
```

Comment:
I haven't seen a `.constants` file before. Is that a common convention?

Reply:
[53e84d8](https://github.com/AMP-SCZ/dpdash/pull/76/commits/53e84d8e5a9cfd0570547f3f451eb6dfe167f766)